docs/contributing/assets/allowed_actions.json

```diff
@@ -98,6 +98,47 @@
       "security_review_performed": true,
       "3rd_party_tool": {},
       "comment": ""
+    },
+    {
+      "name": "ahmadnassri/action-workflow-queue",
+      "versions": [
+        "f547ac848c16a9bb1b2ed4a850e0cc5098af2cf8"
+      ],
+      "repository": "https://github.com/ahmadnassri/action-workflow-queue",
+      "marketplace": "https://github.com/marketplace/actions/workflow-queue",
+      "security_review_performed": true,
+      "3rd_party_tool": {},
+      "comment": ""
+    },
+    {
+      "name": "shmokmt/actions-setup-github-comment",
+      "versions": [
+        "93d9e7c3ea11e473e3061a8dbed8231faea13dea"
+      ],
+      "repository": "https://github.com/shmokmt/actions-setup-github-comment",
+      "marketplace": "https://github.com/marketplace/actions/setup-github-comment",
+      "security_review_performed": true,
+      "3rd_party_tool": {
+        "tool": "suzuki-shunsuke/github-comment",
+        "pinned_version": "v6.0.2",
+        "repository": "https://github.com/golangci/golangci-lint"
```

Comment:
I assume this is the wrong repository (copy/paste error ;-)).

Reply:
Thanks, fixed